framework/framework.go

```diff
@@ -123,39 +122,6 @@ func New(config Config) (*Framework, error) {
 	return f, nil
 }
 
-// AddFunc executes the framework's ProcessCreate function.
-func (f *Framework) AddFunc(obj interface{}) {
```

Comment:
The add function is not executed anymore. We run everything with the update function.